src/App.jsx

```diff
@@ -1,33 +1,48 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
+export const App = () => {
+  const [query, setQuery] = useState('');
+  const [visibleMovies, setVisiblesMovies] = useState(moviesFromServer);
+  const handleQueryChange = event => setQuery(
+    event.target.value
+      .trim()
+      .toLowerCase(),
+  );
 
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+  return (
+    <div className="page">
+      <div className="page-content">
+        <div className="box">
+          <div className="field">
+            <label htmlFor="search-query" className="label">
+              Search movie
+            </label>
+
+            <div className="control">
+              <input
+                onChange={event => handleQueryChange(event)}
+                type="text"
+                id="search-query"
+                className="input"
+                placeholder="Type search word"
```

Comment:
Please fix the English :)

```suggestion
                placeholder="Search"
```


```suggestion
                placeholder="Type to start search"
```

Reply:
thanks for the comments
the task began to look easier))